projects/Unit/APIApproval.Approve.verified.txt

```diff
@@ -782,6 +782,36 @@ namespace RabbitMQ.Client.Exceptions
         public object Offender { get; }
     }
 }
+namespace RabbitMQ.Client.Framing
```

Comment:
Instead of just bringing this back, could we change the namespace to `RabbitMQ.Client` instead?

Reply:
@lukebakken I like this idea. But this PR is already merged, so we'd need another one 🤷‍♂️

Reply:
Easy-peasy, I'll get it right now in a new PR.